### DISCUSSION TITLE
英文缩写 's 间距过大

### DISCUSSION THREAD

**Discussion by WANG-Guangxin** [YunYouJun/hexo-theme-starry#390] [Q&A 疑难杂症]
英文缩写 's 间距过大，应该是默认字体的问题。

除了换字体，有什么别的方案吗？

1. yunyoujun.cn 的默认字体
![image](https://github.com/YunYouJun/hexo-theme-yun/assets/71872849/424f606e-4533-4c06-b988-d70be5e6d080)

2. butterfly.js.org 的默认字体没问题
![image](https://github.com/YunYouJun/hexo-theme-yun/assets/71872849/afb0f023-f305-4486-b716-d11cfc939828)

**Comment by YunYouJun** [comment]
![image](https://github.com/YunYouJun/hexo-theme-yun/assets/25154432/3b131edd-a09e-40b5-abf9-cb9fdc705c1e)

看起来我并没有复现你的问题。
你能确认你的显示字体是什么吗？
